integration-tests/e2e/kubetest/description/1.20/skip.json

```diff
@@ -1,101 +1,12 @@
-[{ "testcase": "[Flaky]"},
-  { "testcase": "Alpha"},
-  { "testcase": "[Experimental]"},
-  { "testcase": "[Driver: vSphere]"},
-  { "testcase": "[Driver: cinder]"},
-  { "testcase": "[Driver: gluster]"},
```

Comment:
see comment above

Reply:
Our openstack provisioner is cinder, hence we could remove the cinder skip to have these test executed.
We can safely remove gluster however as we don't run/deploy it.

Reply:
if I understand correctly remove `cinder` and keep `gluster` in skip is that correct ?

Reply:
yes (removing cinder from skip will result in executing cinder tests -> we want that as we do have cinder csi in place; keeping gluster in skip will result in skipping gluster tests -> we want that as we have no interest in gluster)